upgrade/upd-2.3.3-to-2.4.0/index.php

```
@@ -63,18 +67,48 @@ public function apply_version()
      */
     public function xoops_putLicenseKey($system_key, $licensefile, $license_file_dist = 'license.dist.php')
     {
-        chmod($licensefile, 0777);
-        $fver     = fopen($licensefile, 'w');
         $fver_buf = file($license_file_dist);
+        if (false === $fver_buf) {
+            return false;
+        }
+
+        $content = '';
         foreach ($fver_buf as $line => $value) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$line'. (undefined)

(UnusedLocalVariable)